### PR TITLE
Renamed properties to attributes in Machine model object

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Renames properties to attributes in Machine model object.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/6803

### CODE REVIEW

**Review by tolusha — APPROVED** [review]

**Review by skabashnyuk — APPROVED** [review]

**Review by benoitf — APPROVED** [review]

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/macro/DevMachineHostNameMacro.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public String getDescription() {
     Optional<MachineImpl> devMachine = wsAgentServerUtil.getWsAgentServerMachine();
 
     if (devMachine.isPresent()) {
-      String hostName = devMachine.get().getProperties().get("config.hostname");
```
Comment (garagatyi) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/macro/DevMachineHostNameMacro.java:63]: This code can't be used anymore- we don't have such a property anymore. @vparfonov Please take a look and remove it on an occasion.

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/processes/panel/ProcessesPanelPresenter.java [hunk 2]
```diff
@@ -501,7 +501,7 @@ public void onPreviewSsh(String machineId) {
 
     // user
     final String userName;
-    String user = machine.get().getProperties().get("config.user");
```
Comment (garagatyi) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/processes/panel/ProcessesPanelPresenter.java:504]: @vparfonov see my comment [here](https://github.com/eclipse/che/pull/7687/files#r154576015)

File: plugins/plugin-gdb/che-plugin-gdb-ide/src/main/java/org/eclipse/che/plugin/gdb/ide/configuration/GdbConfigurationPagePresenter.java [hunk 3]
```diff
@@ -109,7 +109,7 @@ public void go(AcceptsOneWidget container) {
   private void setHosts(List<MachineImpl> machines) {
     Map<String, String> hosts = new HashMap<>();
     for (MachineImpl machine : machines) {
-      String host = machine.getProperties().get("network.ipAddress");
```
Comment (garagatyi) [inline plugins/plugin-gdb/che-plugin-gdb-ide/src/main/java/org/eclipse/che/plugin/gdb/ide/configuration/GdbConfigurationPagePresenter.java:112]: @vparfonov see my comment [here](https://github.com/eclipse/che/pull/7687/files#r154576015)

File: plugins/plugin-java/che-plugin-java-ext-lang-server/src/test/java/org/eclipse/che/plugin/java/server/jdt/quickfix/AssistQuickFixTest.java [hunk 4]
```diff
@@ -696,12 +696,12 @@ public void testAssignToLocal10() throws Exception {
     buf.append("package test1;\n");
     buf.append("public class E {\n");
     buf.append("    public void foo() {\n");
-    buf.append("        System.getProperties()\n");
```
Comment (garagatyi) [inline plugins/plugin-java/che-plugin-java-ext-lang-server/src/test/java/org/eclipse/che/plugin/java/server/jdt/quickfix/AssistQuickFixTest.java]: @sleshchenko looks like this class is changed by mistake

File: plugins/plugin-java/che-plugin-java-ext-lang-server/src/test/java/org/eclipse/che/plugin/java/server/jdt/quickfix/ReturnTypeQuickFixTest.java [hunk 5]
```diff
@@ -632,7 +632,7 @@ public void testConstructorReturnsValue() throws Exception {
     buf.append("package test1;\n");
     buf.append("public class E {\n");
     buf.append("    public E() {\n");
-    buf.append("        return System.getProperties();\n");
```
Comment (garagatyi) [inline plugins/plugin-java/che-plugin-java-ext-lang-server/src/test/java/org/eclipse/che/plugin/java/server/jdt/quickfix/ReturnTypeQuickFixTest.java]: @sleshchenko looks like this class is changed by mistake

File: wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/agent/server/WsAgentPingRequestFactoryTest.java [hunk 6]
```diff
@@ -49,7 +49,7 @@
   //
   //        when(httpJsonRequestFactory.fromUrl(anyString())).thenReturn(httpJsonRequest);
   //        when(httpJsonRequest.setMethod(HttpMethod.GET)).thenReturn(httpJsonRequest);
-  //        when(server.getProperties()).thenReturn(serverProperties);
```
Comment (garagatyi) [inline wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/agent/server/WsAgentPingRequestFactoryTest.java]: @vparfonov can you take a look whether we still need this class?

Comment (vparfonov) [reply wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/agent/server/WsAgentPingRequestFactoryTest.java]: we can remove it

File: plugins/plugin-java/che-plugin-java-ext-lang-server/src/test/java/org/eclipse/che/plugin/java/server/jdt/quickfix/AssistQuickFixTest.java (hunk 4 above)
Comment (sleshchenko) [reply plugins/plugin-java/che-plugin-java-ext-lang-server/src/test/java/org/eclipse/che/plugin/java/server/jdt/quickfix/AssistQuickFixTest.java]: Good catch!

File: wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/agent/server/WsAgentPingRequestFactoryTest.java (hunk 6 above)
Comment (sleshchenko) [reply wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/agent/server/WsAgentPingRequestFactoryTest.java]: I've removed it

**Review by garagatyi — APPROVED** [review]

**Pull request merged**